CheatSheets/JavaScript-CheatSheet.md

```diff
@@ -90,7 +90,9 @@ var names= ["Raj", "Ram", "Sham"];
 | splice()      | Adds elements in a specified way and position                           |
 | toString()    | Converts elements to strings                                            |
 | unshift()     | Adds a new element to the beginning                                     |
-| valueOf()     | Returns the primitive value of the specified object                     |
```

Comment:
You deleted the `valueOf()` column. Please, add this line or tell me why you deleted it.